odoo/addons/base/tests/test_res_users.py

```diff
@@ -520,3 +520,16 @@ def test_revoke_all_devices(self):
 
         # In addition, the password must have been emptied from the wizard
         self.assertFalse(user_identity_check.password)
+
+
+@tagged('post_install', '-at_install')
+class TestUsersSessions(UsersCommonCase):
```

Comment:
The unit test cannot be added into an existing test class ? e.g. `class TestUsers` ?